README.md

```diff
@@ -121,6 +121,16 @@ With this enabled you can use `<prefix> C-l` to clear the screen.
 
 Thanks to [Brian Hogan][] for the tip on how to re-map the clear screen binding.
 
+#### Intellectual Clear Screen
+
+Additionally, it is possible to make `<Ctrl-l>` to work as usual if current pane is full-width (this will also work well if that pane is Vim):
+
+``` tmux
+single_pane='test $(tmux list-panes | grep "(active)" | grep -o "\[[0-9]*x" | tr -d "[x") -eq $(tmux list-windows | grep "(active)" | grep -o " \[[0-9]*x" | tr -d "[x")'
+...
+bind -n C-l if-shell "$single_pane || $is_vim" "send-keys C-l" "select-pane -R"
```

Comment:
Great, thanks! (I am a newbie with tmux..)
11 мая 2016 г. 0:14 пользователь "Daniel Hahler" <notifications@github.com>
написал:

> In README.md
> <https://github.com/christoomey/vim-tmux-navigator/pull/126#discussion_r62753664>
> :
>
> > @@ -121,6 +121,16 @@ With this enabled you can use `<prefix> C-l` to clear the screen.
> >
> >  Thanks to [Brian Hogan][] for the tip on how to re-map the clear screen binding.
> >
> > +#### Intellectual Clear Screen
> > +
> > +Additionally, it is possible to make `<Ctrl-l>` to work as usual if current pane is full-width (this will also work well if that pane is Vim):
> > +
> > +``` tmux
> > +single_pane='test $(tmux list-panes | grep "(active)" | grep -o "\[[0-9]*x" | tr -d "[x") -eq $(tmux list-windows | grep "(active)" | grep -o " \[[0-9]*x" | tr -d "[x")'
> > +...
> > +bind -n C-l if-shell "$single_pane || $is_vim" "send-keys C-l" "select-pane -R"
>
> Nice, but it can be made easier:
>
> bind -n C-l if '[ #{pane_width} = #{window_width} ]' "send-keys C-l"
> "select-pane -R"
>
> (This also includes window_zoomed_flag==1.)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/christoomey/vim-tmux-navigator/pull/126/files/bc9091c07e473d4c40ee1abc8492ebe8761ef90f#r62753664>
>